build.gradle

```diff
@@ -76,7 +76,6 @@ subprojects {
             failOnVersionConflict()
 
             def provided = [["org.apache.httpcomponents", "httpclient"],
-                            ["com.google.guava", "guava"],
                             ["com.google.api-client", "google-api-client"]]
```

Comment:
isn't this still provided by the intellij libs?

Reply:
I believe the reason this was added in was to prevent a classpath loading problem when intellij loads from https://github.com/JetBrains/intellij-community/tree/master/lib but they also seem to be on guava 21.

Reply:
Also I think this check was to prevent **any library** from pulling in the transitive `guava`. So this might lead to other dependency clash issues that we might be unaware of.

Reply:
Also I think that link to IJ you put is ahead of whats released. I'm seeing guava 19 being pulled in by the IJ 2017.1 distribution.

I'm open to other approaches to solve this problem. The problem being that the GCS client lib requires guava 20.